### DISCUSSION TITLE
Awesome Quarto list (2023-08-06)

### DISCUSSION THREAD

**Discussion by mcanouil** [quarto-dev/quarto-cli#1610] [Show and Tell]
I started to work on an [Awesome](https://github.com/sindresorhus/awesome) Quarto list: https://github.com/mcanouil/awesome-quarto

I already added some contents mostly based on the official ones and planned to add more in the coming days/weeks such as the recordings showcasing Quarto, etc.

Any contribution welcome!

---

Source: <https://github.com/mcanouil/awesome-quarto#featured-new-releases>

# New Additions (2023-08-06)

- *__Tutorials & workshops__*
  - [Tutorial: Creating Quarto Journal Article Templates](https://christophertkenny.com/posts/2023-07-01-creating-quarto-journal-articles/) - An in-depth blog post detailing the process for converting journal LaTeX templates into Quarto templates.
  - [Tutorial: Personal Website using Jupyter Notebook and Quarto](https://adtarie.net/posts/007-quarto-python-tutorial/) - A Python-oriented step-by-step tutorial on how to create a website using Quarto.
- *__Blog posts__*
  - [Making Pretty PDFs with Quarto](https://nrennie.rbind.io/blog/making-pretty-pdf-quarto/) - A blog post showing how to customise the styling of PDF documents, and save the styling into a Quarto extension to make it easier to reuse and share.
  - [How to self-publish a technical book on Leanpub and Amazon using Quarto](https://www.brodrigues.co/blog/2023-06-29-book_quarto/) - This blog post explains which settings to use to compile an Epub for Leanpub and a print-ready PDF for Amazon's self-publishing service (KDP).
- *__Libraries/Packages/Scripts__*
  - [babelquarto](https://docs.ropensci.org/babelquarto/) - R package to help set up, and render, multilingual Quarto books (see also [babeldown](https://docs.ropensci.org/babeldown/articles/quarto.html).
- *__Extensions__*
  - [version-badge](https://github.com/mcanouil/quarto-version-badge) - An extension for Quarto to provide a shortcode to display software version.
  - [preview-colour](https://github.com/mcanouil/quarto-preview-colour) - An extension to add preview colour as a coloured symbol next to colour code.
  - [lua-env](https://github.com/mcanouil/quarto-lua-env) - An extension for Quarto to provide access to LUA objects as metadata.
  - [spotlight](https://github.com/mcanouil/quarto-spotlight) - An extension for Reveal.js allowing to highlight the current mouse position with a spotlight.
- *__Templates__*
  - [JOAS](https://github.com/espinielli/joas-template) - Quarto journal template for *Journal of Open Aviation Science* (JOAS).
  - [PNAS](https://github.com/christopherkenny/pnas) - Quarto template for *PNAS*.
  - [APSR](https://github.com/christopherkenny/apsr) - Quarto template for *American Political Science Review* (APSR).
  - [cinco-de-mayo](https://github.com/EmilHvitfeldt/quarto-revealjs-cinco-de-mayo) - Cinco de Mayo Quarto `Reveal.js` theme.
  - [rvalhub](https://github.com/pharmaR/rvalhub-quarto) - R Validation Hub `Reveal.js` theme.
- *__Real Life Examples__*
  - [Quering with PRQL](https://github.com/eitsupi/querying-with-prql) - Docusaurus website using computations via `Jupyter` and `knitr` and multiple languages (PRQL, SQL, R, Python, *etc.*).
  - [Real World Data Science](https://realworlddatascience.net/) - The Royal Statistical Society website, built with Quarto, features a custom design (based on the Lux Bootswatch theme), with a customised navbar and homepage layout.
  - [Continental-scale biodiversity data assessment using the Atlas of Living Australia](https://github.com/shandiya/VicBioCon2023) - Slides for éVictorian Biodiversity Conference 2023" (See slides [here](https://shandiya.quarto.pub/vicbiocon2023/)).

**Comment by harrelfe** [comment]
The e-book R Workflow is written in `Quarto` and has a lot of `Quarto` how-tos: https://hbiostat.org/rflow
